CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+- Reporting correct SQS entry span time
```

Comment:
Actually, the span's _duration_ is what has been fixed, not the time 😝

Reply:
"ja ja" I will fix it :D ty!

Reply:
Ja, ja! https://www.youtube.com/watch?v=qrZxw5vdwRI&t=19s